src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends AbstractFigure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculatedArea() {
+        return Math.PI * radius * radius;
+    }
```

Comment:
There is a typo in the method name 'calculatedArea'. It should be 'calculateArea' to follow standard naming conventions and to match the expected method name in the Figure interface.